acceptance/config/common/options.rb

```diff
@@ -0,0 +1,16 @@
+{
+  :ssh                         => {
+    :keys => ["id_rsa_acceptance", "#{ENV['HOME']}/.ssh/id_rsa-acceptance"],
+  },
+  :load_path                   => './lib',
+  :xml                         => true,
+  :timesync                    => false,
+  :repo_proxy                  => true,
+  :add_el_extras               => true,
+  :forge_host                  => 'forge-aio01-petest.puppetlabs.com',
+  :'master-start-curl-retries' => 30,
+  :is_puppetserver             => true,
+  :'use-service'               => true,
+  :puppetservice               => 'puppetserver',
+  :'puppetserver-confdir'      => '/etc/puppetlabs/puppetserver/conf.d',
```

Comment:
Previously, `git`-style tested against webrick, but now we'll be testing against `puppetserver`. Does `git` still work?